spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/services/SCConfigService.java

```diff
@@ -123,7 +123,7 @@ private void loadConfigs(List<File> configFiles) {
         registerForms(configFiles);
         String remoteUrl = getRemoteConfigUrl(configFiles);
         if (remoteUrl != null) {
-            loadRemoteConfig(remoteUrl);
+            loadRemoteConfig(remoteUrl + "/config");
```

Comment:
Make this check the end of remoteUrl for a trailing slash and only add one if needed